docs/src/main/asciidoc/websockets.adoc

```diff
@@ -162,3 +162,79 @@ As usual, the application can be packaged using `./mvnw clean package` and execu
 You can also build the native executable using `./mvnw package -Pnative`.
 
 You can also test your web socket applications using the approach detailed {quickstarts-blob-url}/websockets-quickstart/src/test/java/org/acme/websockets/ChatTest.java[here].
+
+== Weboscket Clients
```

Comment:
typo here

Reply:
Good catch! Would you like to provide a PR to fix it?

Reply:
https://github.com/quarkusio/quarkus/pull/12934